website/src/_posts/2021-08-2.0.md

```diff
@@ -34,16 +34,19 @@ With that in mind, we felt it was high time to give Uppy some more much-needed t
 
 ## Table of Contents
 
-- [Highlights since 1.0](#Highlights-since-1-0)
-- [Smaller bundles](#Smaller-bundles)
-- [Faster](#Faster)
-- [Preact X and upgraded dependencies](#Preact-X-and-upgraded-dependencies)
-- [`Plugin` is replaced with `BasePlugin` and `UIPlugin`](#Plugin-is-replaced-with-BasePlugin-and-UIPlugin)
-- [Strict TypeScript types by default](#Strict-TypeScript-types-by-default)
-- [Batch pre-signing URL's for AWS S3 Multipart](#Batch-pre-signing-URLs-for-AWS-S3-Multipart)
-- [And more](#And-more)
-- [What future remains for 1.0?](#What-future-remains-for-1-0?)
-- [That's it](#That’s-it)
+- [A pup no more](#a-pup-no-more)
```

Comment:
These unrelated changes can be reverted

Reply:
oh odd I'm not sure how this got in. Let me remove the change 😄